third_party/musl/BUILD.gn

```diff
@@ -0,0 +1,519 @@
+    "src/string/memmem.c",
+    "src/string/memmove.c",
+    "src/string/mempcpy.c",
+    "src/string/memrchr.c",
+    "src/string/memset.c",
+    "src/string/rindex.c",
+    "src/string/stpcpy.c",
+    "src/string/stpncpy.c",
+    "src/string/strcat.c",
+    "src/string/strchr.c",
+    "src/string/strchrnul.c",
+    "src/string/strcmp.c",
+    "src/string/strcpy.c",
+    "src/string/strcspn.c",
+    "src/string/strerror_r.c",
+    "src/string/strlcat.c",
+    "src/string/strlcpy.c",
+    "src/string/strlen.c",
+    "src/string/strncat.c",
+    "src/string/strncmp.c",
+    "src/string/strncpy.c",
+    "src/string/strnlen.c",
+    "src/string/strpbrk.c",
+    "src/string/strrchr.c",
+    "src/string/strsep.c",
+    "src/string/strspn.c",
+    "src/string/strstr.c",
+    "src/string/strtok.c",
+    "src/string/strtok_r.c",
+    "src/string/strverscmp.c",
+    "src/string/swab.c",
+    "src/string/wcpcpy.c",
+    "src/string/wcpncpy.c",
+    "src/string/wcscat.c",
+    "src/string/wcschr.c",
+    "src/string/wcscmp.c",
+    "src/string/wcscpy.c",
+    "src/string/wcscspn.c",
+    "src/string/wcslen.c",
+    "src/string/wcsncat.c",
+    "src/string/wcsncmp.c",
+    "src/string/wcsncpy.c",
+    "src/string/wcsnlen.c",
+    "src/string/wcspbrk.c",
+    "src/string/wcsrchr.c",
+    "src/string/wcsspn.c",
+    "src/string/wcsstr.c",
+    "src/string/wcstok.c",
+    "src/string/wcswcs.c",
+    "src/string/wmemchr.c",
+    "src/string/wmemcmp.c",
+    "src/string/wmemcpy.c",
+    "src/string/wmemmove.c",
+    "src/string/wmemset.c",
+    "src/time/__month_to_secs.c",
+    "src/time/__tm_to_secs.c",
+    "src/time/__year_to_secs.c",
+    "src/time/strftime.c",
+  ]
+
+  if (musl_arch == "i386" || musl_arch == "x86_64" || musl_arch == "aarch64") {
+    sources += [
+      "src/fenv/$musl_arch/fenv.s",
+      "src/setjmp/$musl_arch/longjmp.s",
+      "src/setjmp/$musl_arch/setjmp.s",
+    ]
+  }
+  if (musl_arch == "arm") {
+    sources += [
+      "src/fenv/$musl_arch/fenv-hf.S",
+      "src/fenv/$musl_arch/fenv.c",
+      "src/setjmp/$musl_arch/longjmp.s",
+      "src/setjmp/$musl_arch/setjmp.s",
+    ]
+  }
+  if (musl_arch == "mips") {
+    sources += [
+      "src/fenv/$musl_arch/fenv-sf.c",
+      "src/fenv/$musl_arch/fenv.S",
+      "src/setjmp/$musl_arch/longjmp.S",
+      "src/setjmp/$musl_arch/setjmp.S",
+    ]
+  }
+  if (musl_arch == "i386" || musl_arch == "x86_64") {
+    sources += [
+      "src/string/$musl_arch/memcpy.s",
+      "src/string/$musl_arch/memmove.s",
+      "src/string/$musl_arch/memset.s",
+    ]
+    sources -= [
+      "src/string/memcpy.c",
+      "src/string/memmove.c",
+      "src/string/memset.c",
+    ]
+  }
+
+  deps = [
+    "//starboard:starboard_headers_only",
+    "//starboard/common:common_headers_only",
+  ]
+}
+
+target(gtest_target_type, "musl_unittests") {
+  testonly = true
+  sources = [ "test/type_size_test.cc" ]
+  deps = [
+    ":c",
+    "//starboard:starboard_headers_only",
+  ]
+}
```

Comment:
missing endline .. not sure why pre-commit isn't catching it ? you can run it on a single commit if you need with `pre-commit run --from-ref=<hash> --to-ref=HEAD`